### DISCUSSION TITLE
Allow Merging Multi-Language Sites

### DISCUSSION THREAD

**Discussion by Unkn0wnCat** [plausible/analytics#1653] [Anything Else]
Hi all,

I'm using Plausible for my personal website at https://kevink.dev and I love it. However the site branches out to https://kevink.dev/en/ for the English version and https://kevink.dev/de/ for the German one.

This means that in Plausible I always see separate entries for each page on my site, one prefixed with /en/ and one with /de/.

You can see the problem in action at https://analytics.kevink.dev/kevink.dev.

I wish plausible had an option to specify a "multi-language-mode", which would only affect display of the data. The multi language modes could be:
1. Merging subpaths (e.g. /blog, /de/blog, /ru/blog => /blog)
2. Merging subdomains (e.g. example.com/blog, de.example.com/blog => /blog)
3. Merging domains (e.g. example.com/blog, beispiel.de/blog => /blog)

For 2 and 3 this could be done via a "virtual" site, which merges display of two or more different sites.

Can't wait to think what you all think of this!

Have a nice day,
Kevin

**Comment by metmarkosaric** [comment]
thanks for the feedback @Unkn0wnCat! there's another feature request that would help you with this use case as well. see https://github.com/plausible/analytics/discussions/320

**Reply by Unkn0wnCat** [reply]
#320 is close, but does not offer merging the views. As far as I can see the custom dimensions would allow me to attach the language to the clicks, but that only means that I could filter by language, not that I can merge /en/page and /de/page into one when viewing them. 

My request is not about attaching additional data, but about extracting the language from the existing paths. 

Please correct me if I missed something

**Reply by metmarkosaric** [reply]
have you tried our filtering for that as it allows you to group pages such as /en/page and /de/page already? see https://plausible.io/docs/filters-segments#using-the-add-filter-menu-for-extra-functionality

**Reply by Unkn0wnCat** [reply]
The filters unfortunately only allow for viewing one page-pair at a time. I would like to be able to view which ones are my top pages regardless of the language prefix.

Example for Top Pages:
```
/en/       24
/de/       16
/de/blog/  14
/en/about/ 13
/de/about/ 11
/en/blog/   4
```
Would become
```
/          40
/about/    24
/blog/     18
```

This would allow for analyzing the popularity of subpages regardless of language, but it is currently not possible inside Plausible without exporting and calculating externally.

**Reply by Vigasaurus** [reply]
You could also try setting up page visit goals for each of the pages you care about (as a alternative for now) - i.e. set up page view goals for `/*/blog` and `/*/about`, which you can then directly compare. It's not necessarily ideal if you have a lot of pages to do this for, but it will let you quickly see the data you want.

**Reply by metmarkosaric** [reply]
thanks @Vigasaurus! the docs for pageview goals are https://plausible.io/docs/pageview-goals

**Reply by Unkn0wnCat** [reply]
@Vigasaurus Thanks, that is a workaround that may work for some cases. However this also breaks the actual goal percentages and as you said it is not ideal for many pages. This is why I will be holding off for now.

I don't actually think a simple "merged display" would be that hard to do, so this would still be my preferred solution.
